connectors/googlemusic.js

```diff
@@ -6,6 +6,17 @@ new class extends BaseConnector {
 
         this.name = 'Google Play Music';
         this.prefix = '/com/google';
+
+        this.artistsSelector = '#player-artist';

```

Comment:
Isn't `artistSelector` equivalent to removed `get artist()`?

And this is element ID, so there shouldn't be more then one element.